serving/setting-up-custom-ingress-gateway.md

```diff
@@ -0,0 +1,256 @@
+# Setting Up Custom Ingress Gateway
```

Comment:
@dgerd Can you please file a bug regarding the custom ingressgateway instruction and assign to me?  These instructions may be a bit out-dated.  Istio now provides Helm template to do this which is much less error-proned and will match with the Istio version the user may be using.

Thasnk

Reply:
I will follow up with a PR for gateway instruction.

Reply:
Created #875 to track this issue.

I would still like to get this change as this enables us to delete the duplicated and also outdated documentation in the knative/serving tree.

Reply:
Can you please add a TODO link to #875 in the doc here?  Thanks

Reply:
done